src/modules/Server/public/Get-SdnServerConfigurationState.ps1

```diff
@@ -85,6 +82,7 @@ function Get-SdnServerConfigurationState {
         Get-SdnVMNetworkAdapterPortProfile -AllVMs | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-SdnVMNetworkAdapterPortProfile' -FileType txt -Format Table
         Get-VMNetworkAdapterIsolation | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-VMNetworkAdapterIsolation' -FileType txt -Format Table
         Get-VMNetworkAdapterRoutingDomainMapping | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-VMNetworkAdapterRoutingDomainMapping' -FileType txt -Format Table
+        Get-VMSystemSwitchExtensionPortFeature -FeatureId "9940cd46-8b06-43bb-b9d5-93d50381fd56" | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'Get-VMSystemSwitchExtensionPortFeature' -FileType json
 
```

Comment:
Is this duplicate with Get-SdnVMNetworkAdapterPortProfile?

Reply:
Slightly different although related. This cmdlet will show the currently enabled extensions on the VMSwitch. This is useful to ensure that VFP Switch Extension is enabled/disabled on VMSwitch. The `Get-SdnVMNetworkAdapterPortProfile` cmdlet will show the profile applied to the VM Network Adapter for the given virtual machine(s).